apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -210,7 +210,8 @@ class ShareAllowedDialog extends React.Component {
       embedOptions = {
         // If you change this width and height, make sure to update the
         // #visualizationColumn.wireframeShare css
-        iframeHeight: applabConstants.APP_HEIGHT + 140,
+        // Extra 40 pixels added to account for left and right padding divs (20 px each side)
+        iframeHeight: applabConstants.APP_HEIGHT + 140 + 40,
```

Comment:
maybe i'm missing something, but why are we changing `iframeHeight` instead of `iframeWidth` here?

Reply:
also, there's a comment above that seems relevant:

```javascript
// If you change this width and height, make sure to update the
// #visualizationColumn.wireframeShare css 
```

Reply:
Ah whoops, when refactoring, I added the pixels to the wrong height/width, so I'll need to go ahead and switch that back.

Reply:
As far as the comment, I tried to address that in the PR description: `Because this is scoped to the embedded share view, I can ignore the comment that says changes in width should also be added to visualizationColumn.wireframeShare.` This issue is limited to the embed share view, but not the traditional share page. The css in the visualizationColumn.wireframeShare keeps track of the width of the visualization. This embedded width calculation is used to determine how big the embedded iframe should be. So the phone visualization width stays the same, but since the embedded iframe holds the phone visualization and the spacing divs, we calculate the embedded iframe width using both the width of the phone visualization and the spacing divs. This comment is from many years ago, so there's not a lot of context on it, but this is my understanding based on some GitHub archaeology. It's hard for me to describe without showing you the different divs, so let me know if you want to hop on a video call!

Reply:
ah, sorry, i missed that detail in your PR description. i mostly understand, but can you point me to where/how the "spacing divs" are rendered? i'm wondering if we have this same problem in gamelab, but i don't quite understand where the spacing divs come into play.

Reply:
"containerLeft", for spacing: https://github.com/code-dot-org/code-dot-org/blob/60c1c5e6133ddf213530035a436f13dd10901fcc/apps/src/StudioApp.js#L2281

As far as whether this affects Game Lab, the answer is . . . yes? But also no. At some point along the way, the phone frame was removed from the Game Lab share visualization, but the calculated embed width (https://github.com/code-dot-org/code-dot-org/blob/60c1c5e6133ddf213530035a436f13dd10901fcc/apps/src/code-studio/components/ShareAllowedDialog.jsx#L221), was never updated to remove the 32 pixels that are added to account for the frame. So, because we add the 32 pixels, the visualization is not cut off (although part of the 20px spacing div on the right is). So it's in the same situation, but didn't have the bug that AppLab did.

Reply:
I'm going to update the offset for GameLab to be 4o pxls so that it's clear how that lines up with the AppLab one.